src/app/content/studyGuides/components/Filters.tsx

```diff
@@ -119,7 +122,11 @@ export default () => {
         <UsingThisGuideButton onClick={toggleUsingThisGuide} open={isUTGopen}/>
       </RightButtonsWrapper>
     </FiltersTopBar>
-    { isUTGopen && <UsingThisGuideBanner onClick={toggleUsingThisGuide} isUTGopen={isUTGopen}/> }
+    <UsingThisGuideBanner
+      onClick={toggleUsingThisGuide}
+      show={isUTGopen}
+      isOpenedForTheFirstTime={showUTGInitially}
+    />
```

Comment:
I think it will  only need the `onClick` prop if its refactored

Reply:
https://github.com/openstax/rex-web/pull/772#discussion_r479855161